tests/test_checks.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
-from tempfile import NamedTemporaryFile
 import errno
-from unittest2 import TestCase
+from tempfile import NamedTemporaryFile
+from unittest import TestCase
 
 from mock import patch
 
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
-                         FilesExistHealthCheck, FilesDontExistHealthCheck)
+from healthcheck.checks import (
```

Comment:
I101 Imported names are in the wrong order. Should be FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck

Reply:
I think Snitch is right here.